zulipterminal/helper.py

```diff
@@ -152,9 +151,10 @@ def set_count(id_list: List[int], controller: Any, new_count: int) -> None:
                                                    new_count)
                         break
             # FIXME: Update unread_counts['unread_topics']?
-            if ([messages[id]['display_recipient'], msg_topic] in
-                    controller.model.muted_topics):
-                add_to_counts = False
+            # FIXME
+            #if ([messages[id]['display_recipient'], msg_topic] in
+                    #controller.model.muted_topics):
+                #add_to_counts = False
```

Comment:
What does display_recipient mean?

Reply:
`display_recipient` has the intended recipient of the message. In case of messages to stream this will be the stream name. If to a PM it will be things like User ID, email, full_name, etc. 

Try grepping `display_recipient` in our code-base and read through the tests. You'll get examples of how display_recipients can take on different forms.